care/facility/api/viewsets/patient_sample.py

```diff
@@ -3,15 +3,19 @@
 from django_filters import rest_framework as filters
 from dry_rest_permissions.generics import DRYPermissionFiltersBase, DRYPermissions
 from rest_framework import viewsets
+from rest_framework.decorators import action
```

Comment:
Codacy found an issue: [Unable to import 'rest_framework.decorators'](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5838481)